base_product_mass_addition/readme/ROADMAP.rst

```diff
@@ -0,0 +1,2 @@
+Adding new implementations would be great:
+on sale.order or on stock.picking.batch for instance.
```

Comment:
https://github.com/OCA/sale-workflow/tree/14.0/sale_quick exist for now

and also

https://github.com/acsone/stock-logistics-workflow/tree/14.0/stock_picking_quick

Then ROADMAP.rst may be deleted I suppose

Reply:
(and moved into the description part).